Initial setup: Could not resolve "tailwindcss/plugin"

```
$ mix setup
[...]

$ mix assets.build

Rebuilding...

Done in 156ms.

  ../priv/static/assets/app.js  232.8kb

⚡ Done in 14ms
✘ [ERROR] Could not resolve "tailwindcss/plugin"

    tailwind.config.js:1:23:
      1 │ const plugin = require("tailwindcss/plugin")
        ╵                        ~~~~~~~~~~~~~~~~~~~~

  You can mark the path "tailwindcss/plugin" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.

✘ [ERROR] Could not resolve "@tailwindcss/forms"

    tailwind.config.js:17:12:
      17 │     require("@tailwindcss/forms"),
         ╵             ~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@tailwindcss/forms" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.

✘ [ERROR] Could not resolve "@tailwindcss/typography"

    tailwind.config.js:18:12:
      18 │     require("@tailwindcss/typography"),
         ╵             ~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@tailwindcss/typography" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.

3 errors
** (Mix) `mix esbuild tailwind_bundle` exited with 1
```

Running in a [devenv](https://devenv.sh) environment:

```nix
{ pkgs, lib, config, inputs, ... }:

{
  # https://devenv.sh/packages/
  packages =  
    # Darwin only
    with pkgs; lib.optionals pkgs.stdenv.isDarwin [
      # for ExUnit notifier
      terminal-notifier

      # for package - file_system
      darwin.apple_sdk.frameworks.CoreFoundation
      darwin.apple_sdk.frameworks.CoreServices
    ];

  # https://devenv.sh/languages/
  languages.elixir.enable = true;

  # https://devenv.sh/services/
  services.postgres = {
    enable = true;
    port = 5533;
    initialScript = ''
      CREATE ROLE dev WITH LOGIN PASSWORD 'dev' SUPERUSER;
    '';
  };

  # See full reference at https://devenv.sh/reference/options/
}
```

Then I installed npm via devenv and ran `npm install --prefix assets`, that did help.

```
$ mix assets.build

Rebuilding...

Done in 157ms.

  ../priv/static/assets/app.js  232.8kb

⚡ Done in 9ms

  ../priv/tailwind.config.bundle.js  272.2kb

⚡ Done in 6ms

$ mix assets.deploy

Rebuilding...

Done in 237ms.

  ../priv/static/assets/app.js  111.4kb

⚡ Done in 9ms

  ../priv/tailwind.config.bundle.js  124.9kb

⚡ Done in 5ms
Compiling 15 files (.ex)
    warning: Heroicons.arrow_path/1 is undefined (module Heroicons is not available or is yet to be defined)
    │
 45 │       Attempting to reconnect <Heroicons.arrow_path class="ml-1 w-3 h-3 inline animate-spin" />
    │                                          ~
    │
    └─ lib/beacon_demo_web/components/layouts/app.html.heex:45:42: BeaconDemoWeb.Layouts.app/1

     warning: Heroicons.arrow_left/1 is undefined (module Heroicons is not available or is yet to be defined)
     │
 508 │         <Heroicons.arrow_left solid class="w-3 h-3 stroke-current inline" />
     │                    ~
     │
     └─ lib/beacon_demo_web/components/core_components.ex:508:20: BeaconDemoWeb.CoreComponents."back (overridable 1)"/1

     warning: Heroicons.exclamation_circle/1 is undefined (module Heroicons is not available or is yet to be defined)
     │
 146 │         <Heroicons.exclamation_circle :if={@kind == :error} mini class="h-4 w-4" />
     │                    ~
     │
     └─ lib/beacon_demo_web/components/core_components.ex:146:20: BeaconDemoWeb.CoreComponents."flash (overridable 1)"/1
     └─ lib/beacon_demo_web/components/core_components.ex:379:18: BeaconDemoWeb.CoreComponents."error (overridable 1)"/1

     warning: Heroicons.information_circle/1 is undefined (module Heroicons is not available or is yet to be defined)
     │
 145 │         <Heroicons.information_circle :if={@kind == :info} mini class="h-4 w-4" />
     │                    ~
     │
     └─ lib/beacon_demo_web/components/core_components.ex:145:20: BeaconDemoWeb.CoreComponents."flash (overridable 1)"/1

    warning: Heroicons.x_mark/1 is undefined (module Heroicons is not available or is yet to be defined)
    │
 77 │                   <Heroicons.x_mark solid class="h-5 w-5 stroke-current" />
    │                              ~
    │
    └─ lib/beacon_demo_web/components/core_components.ex:77:30: BeaconDemoWeb.CoreComponents."modal (overridable 1)"/1
    └─ lib/beacon_demo_web/components/core_components.ex:151:20: BeaconDemoWeb.CoreComponents."flash (overridable 1)"/1

Generated beacon_demo app
Check your digested files at "priv/static"
```

Not sure about those warnings...

Congratulations on version 0.1.0, btw!  🙂

--- a/mix.exs
+++ b/mix.exs
@@ -81,8 +81,8 @@ defmodule BeaconDemo.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
-      setup: ["deps.get", "assets.setup", "ecto.setup"],
-      "assets.setup": ["tailwind.install --if-missing", "esbuild.install --if-missing"],
+      setup: ["deps.get", "assets.setup", "assets.build", "ecto.setup"],
+      "assets.setup": ["tailwind.install --if-missing", "esbuild.install --if-missing", "cmd npm install --prefix assets"],
       "assets.build": ["tailwind default", "esbuild default", "esbuild beacon_tailwind_config"],
       "assets.deploy": [
         "tailwind default --minify",

